spyder/widgets/fileswitcher.py

```diff
@@ -302,7 +302,13 @@ def widgets(self):
 
     @property
     def line_count(self):
-        return [widget.get_line_count() for widget in self.widgets]
+        line_count = []
+        for widget in self.widgets:
+            try:
+                line_count.append(widget[0].get_line_count())
```

Comment:
@dalthviz if the possible exception is only triggered by

`widget[0].get_line_count()` then I suggest this is written as:

```python
line_count = []
for widget in self.widgets:
    try:
        current_line_count = widget[0].get_line_count()
    except AttributeError:
        current_line_count = 0
    line_count.append(current_line_count)
```

It is subtle difference, but makes clear what the possible error might be